### PR TITLE
Excluded dates message

### PR DESCRIPTION
**Pull request by dvelazquez1282**
I have added the possibility of add a reason message in each exluded day

### CODE REVIEW

File: yarn.lock
Comment (martijnrusschen) [inline yarn.lock:1]: Same here, this change isn't needed for this PR.

Comment (dvelazquez1282) [reply yarn.lock:1]: Oh I'm sorry. I have reverted the changes

File: src/day.jsx [hunk 1]
```diff
@@ -326,13 +327,23 @@ export default class Day extends React.Component {
 
   // A function to return the holiday's name as title's content
   getTitle = () => {
-    const { day, holidays = new Map() } = this.props;
+    const { day, holidays = new Map(), excludeDates } = this.props;
     const compareDt = formatDate(day, "MM.dd.yyyy");
     if (holidays.has(compareDt)) {
       return holidays.get(compareDt).holidayNames.length > 0
```
[the remark this answers is not in the document]
Comment (dvelazquez1282) [reply src/day.jsx:333]: I'm sorry. I don't understand the question. I didn't any change about the holidays

File: src/date_utils.js [hunk 2]
```diff
@@ -409,7 +409,9 @@ export function isDayDisabled(
   return (
     isOutOfBounds(day, { minDate, maxDate }) ||
     (excludeDates &&
-      excludeDates.some((excludeDate) => isSameDay(day, excludeDate))) ||
+      excludeDates.some((excludeDate) =>
```
[the remark this answers is not in the document]
Comment (dvelazquez1282) [reply src/date_utils.js:412]: I have just added a test for excluded dates

File: src/day.jsx [hunk 3]
```diff
@@ -326,13 +327,23 @@
...
         ? holidays.get(compareDt).holidayNames.join(", ")
         : "";
     }
+    if (this.isExcluded()) {
```
[the remark this answers is not in the document]
Comment (dvelazquez1282) [reply src/day.jsx:336]: done. Thanks for your suggestion

File: src/day.jsx [hunk 4]
```diff
@@ -326,14 +327,24 @@ export default class Day extends React.Component {
...
+    const titles = [];
     if (holidays.has(compareDt)) {
-      return holidays.get(compareDt).holidayNames.length > 0
-        ? holidays.get(compareDt).holidayNames.join(", ")
-        : "";
+      titles.push(...holidays.get(compareDt).holidayNames);
     }
-    return "";
+    if (this.isExcluded()) {
+      if (excludeDates) {
```
[the remark this answers is not in the document]
Comment (dvelazquez1282) [reply src/day.jsx]: true. I just deleted

**Pull request merged**